packages/mqttsuite/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/SNodeC/mqttsuite
+TERMUX_PKG_DESCRIPTION="MQTTSuite: A lightweight MQTT Integration System"
+TERMUX_PKG_LICENSE="MIT OR LGPL-3.0-or-later"
```

Comment:
Multiple licenses should be delimited with a comma.
```suggestion
TERMUX_PKG_LICENSE="MIT, LGPL-3.0-or-later"
```
For example see `ipfs`:
https://github.com/termux/termux-packages/blob/054de3f8fe10c4f51a1a518142fd0981099c704f/packages/ipfs/build.sh#L3